pkg/cluster/internal/providers/podman/util.go

```diff
@@ -102,7 +103,12 @@ func getVolumes(label string) ([]string, error) {
 	// Trim away the last `\n`.
 	trimmedOutput := strings.TrimSuffix(string(output), "\n")
 	// Get names of all volumes by splitting via `\n`.
-	return strings.Split(string(trimmedOutput), "\n"), nil
+	for _, v := range strings.Split(string(trimmedOutput), "\n") {
+		if v != "" {
```

Comment:
it parses the load balancer container and adds an empty volume to the list, we can remove all the empty volumes to prevent the same issue in the future

Reply:
there's definitely a more efficient way to do this, multiple actually.
https://github.com/golang/go/wiki/SliceTricks#filter-in-place

Reply:
done